pyproject.toml

```diff
@@ -8,6 +8,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.10"
 nodestream = "^0.11.0"
+aiobotocore = "^2.9.0"
```

Comment:
For the sake of builds being consistent from change to change, we generally version a lock file (Somewhat weird I know) But do you mind running a `poetry lock` and committing the changes?

Reply:
Added